cwd/app.js

```diff
@@ -245,6 +299,36 @@
 
   };
 
+  const rotateDisplay = views => {
+    const currentView = views[index];
+
+    if (Array.isArray(currentView.gauges)) {
+      // Remove highlight from current gauge.
+      const gauge = document.getElementById(`gauge-${gaugeIndex + 1}`);
+      gauge.classList.remove('currentGauge');
+
+      // Switch to the next gauge.
+      gaugeIndex++;
+
+      if (gaugeIndex === currentView.gauges.length) {
+        gaugeIndex = 0;
+      } else {
+        const gauge = document.getElementById(`gauge-${gaugeIndex + 1}`);
+        gauge.classList.add('currentGauge');
+
+        // Prevent us from running the bottom code of the function
+        // that switches to the next view.
+        return;
```

Comment:
Do you have any ideas for making the control of this function less disgusting?  I wrote it, but just wanted your feedback on it.

Reply:
I could think of a couple alternative ways to deal with them, but I don't know if any of them are really any less "disgusting" than this. What's your issue in particular with the way it is now?

Reply:
The big issue that I have is that we have a `return` statement used to exit in order to avoid wrapping the bottom code in an `if` statement.  I think that this initial code got merged into master already, right?  Just want to confirm that's your understanding too - I can just make a new card for this.

Reply:
I actually don't think this is in master yet. Is this still an issue?

Reply:
I'm not sure.  It's a bit of an aesthetic issue, but maybe it isn't worth refactoring.  My view is simply if you have a decent idea for not making it disgusting, let's make it happen.  But we have bigger things to worry about if it isn't too easy.